sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -40,36 +40,97 @@
 from numba.extending import (overload, overload_method, overload_attribute)
 from sdc.hiframes.pd_dataframe_ext import DataFrameType
 from numba.errors import TypingError
+import sdc.datatypes.hpat_pandas_dataframe_types
 
 from sdc.datatypes.hpat_pandas_series_functions import TypeChecker
 
-def sdc_pandas_dataframe_reduce_columns(df, name, params):
 
-    saved_columns = df.columns
-    data_args = tuple('data{}'.format(i) for i in range(len(saved_columns)))
+def _dataframe_reduce_columns_codegen(func_name, func_params, series_params, columns):
```

Comment:
Can you please add a multi-line comment at the top of this function (not a docstring) with a short example of how func_text will look like (for example for mean and a DF with 1 column)?

Reply:
I'm still seeing here docstring instead of comment